




<?xml version="1.0"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">





<html xmlns="http://www.w3.org/1999/xhtml" lang="ko" xml:lang="ko">
<head>
	<title>3D Interaction Viewer & database</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
	<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
	
	<link href="resources/css/basic.css" rel="stylesheet" type="text/css" charset="utf-8" media="all" />
	
	<link rel="stylesheet" type="text/css" href="resources/css/main.css" />
	
	<script>
	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	
	  ga('create', 'UA-89531783-1', 'auto');
	  ga('send', 'pageview');
	</script>
</head>
<body style='overflow-y:scroll;'>
		<div id="headerTag"></div>
		<div id="sitewrapper" class="sitewrapper">
		<!-- menu S -->
		
			<div id="tmbox" class="tmbox">
				<h2 class="hidden">???</h2>
				
				<ul id="tmenu">
					<!-- <li><a href="/3div/intro">ABOUT</a></li> -->
					<!-- <li><a href="/3div/graph">HI-C</a></li> -->
					

				</ul>
			</div>
	<!-- menu E-->
<!-- </body>
</html>
 -->
 
  
	<script type="text/javascript" src="resources/js/jquery/jquery-3.1.1.min.js"></script>
	<script type="text/javascript" src="resources/js/jquery/jquery-ui.js"></script>
    
<style>
a#MOVE_TOP_BTN {
    position: fixed;
    right: 15%;
    bottom: 200px;
    display: none;
    z-index: 999;
}
</style>

<script>
    $(function() {
    	
        $(".tutorial_btn").click(function() {
        	var id = $(this).attr('id')
        	var split = id.split('tutorial_btn');
            $('html, body').animate({
                scrollTop: $("#tutorial"+split[1]).offset().top
            }, 400);        
        });
    	
        $(window).scroll(function() {
            if ($(this).scrollTop() > 500) {
                $('#MOVE_TOP_BTN').fadeIn();
            } else {
                $('#MOVE_TOP_BTN').fadeOut();
            }
        });
        
        $("#MOVE_TOP_BTN").click(function() {
            $('html, body').animate({
                scrollTop : 0
            }, 400);
            return false;
        });
    });
</script>

<div id="contents" class="subcon conwrap" style='margin-top:50px; line-height: 200%; padding: 0px 20px 20px 20px;'>

	<div class="tutorial">
		<ul>
			<a class='tutorial_btn' id='tutorial_btn1'>1. Identification and Annotation of significant interaction</a>
			<a class='tutorial_btn' id='tutorial_btn1-1'><span style="margin-left:50px;">1.1 Input parameters</span></a>
			<li class='tutorial_btn' id='tutorial_btn1-2'><span style="margin-left:50px;">1.2 Output</span></li>
			<li class='tutorial_btn' id='tutorial_btn1-3'><span style="margin-left:50px;">1.3 Result Adjustment</span></li>
			<li class='tutorial_btn' id='tutorial_btn1-4'><span style="margin-left:50px;">1.4 Result Interpretation</span></li>
			
			<li class='tutorial_btn' id='tutorial_btn2' style='margin-top:10px;'>2. Visualization of chromatin interaction</li>
			<li class='tutorial_btn' id='tutorial_btn2-1'><span style="margin-left:50px;">2.1 Input parameters</span></li>
			<li class='tutorial_btn' id='tutorial_btn2-2'><span style="margin-left:50px;">2.2 Output</span></li>
			<li class='tutorial_btn' id='tutorial_btn2-3'><span style="margin-left:50px;">2.3 Result Adjustment</span></li>
			
			<li class='tutorial_btn' id='tutorial_btn3' style='margin-top:10px;'>3. Comparative visualization of chromatin interaction</li>
			<li class='tutorial_btn' id='tutorial_btn3-1'><span style="margin-left:50px;">3.1 Input parameters</span></li>
			<li class='tutorial_btn' id='tutorial_btn3-2'><span style="margin-left:50px;">3.2 Output</span></li>
			<li class='tutorial_btn' id='tutorial_btn3-3'><span style="margin-left:50px;">3.3 Result Adjustment</span></li>
			<li class='tutorial_btn' id='tutorial_btn3-4'><span style="margin-left:50px;">3.4 Result Interpretation</span></li>
			
			<li class='tutorial_btn' id='tutorial_btn4' style='margin-top:10px;'>4. Session Save/Load and Visualization Export</li>
			<li class='tutorial_btn' id='tutorial_btn4-1'><span style="margin-left:50px;">4.1 Save session as a file</span></li>
			<li class='tutorial_btn' id='tutorial_btn4-2'><span style="margin-left:50px;">4.2 Load session from a file</span></li>
			<li class='tutorial_btn' id='tutorial_btn4-3'><span style="margin-left:50px;">4.3 Export Figures</span></li>
		</ul>
	</div>

 	<h3 id='tutorial1'> Session 1 : Identification and Annotation of significant interaction</h3>

	 	&nbsp;Hi-C, high-throughput chromatin conformation capture (3C), is a most-widely used genome-wide molecular assay that identify long-range chromatin interactions. Genomic and epigenomic contexts may assist functional interpretation of the identified interactions. In this regard, 3DIV provides a list of interaction partners of queried loci together genomic/epigenomic annotations. In this tutorial, we provide an example with rs1421085 SNPs in H1-derived mesenchymal stem cell since this SNP is well characterized by significant interactions with IRX3 and IRX5 gene promoters.<br/><br/>

	<h4 id='tutorial1-1'> - Input parameters</h4>
		1) Click &quot;more experiment&quot; button to load the full list of samples.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_1.png" style='width:600px; '/></div><br/>
		
		2) Select H1-derived Mesenchymal Stem Cell by clicking the checkbox.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_2.png" style='width:600px; '/></div><br/>
		
		3) 3DIV allows users to pass the gene names or SNP ID, as well as genomic coordinates as the query. Type rs1421085 and click &quot;Add Sample(s)&quot; button.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_3.png" style='width:600px; '/></div><br/>
		
		4) Click &quot;rs1421085&quot; to pass rs1421085 variant as the query.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_4.png" style='width:600px; '/></div><br/>
		
		5) Then, click &quot;Run&quot; button.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_5.png" style='width:600px; '/></div><br/>

	<h4 id='tutorial1-2'> - Output</h4>
		&nbsp;The output table contains following information; experimental bias-removed interaction frequency, distance-normalized interaction frequency, annotation of enhancers or super-enhancers, disease associated SNPs, and corresponding gene name for a promoter. Fold changes between FPKM of multiple histone ChIP-seq and input are also provided.
		<br/><br/>&nbsp;Bias-removed interaction frequency indicates experimental bias-removed interaction frequencies from raw Hi-C data, defined as the ratio between observed interaction frequency and expected interaction frequency. We also provide distance-normalized interaction frequencies to identify biologically meaningful interactions with LOESS regression model.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_6.png" style='width:800px; '/></div>
	
	<h4 id='tutorial1-3'> - Result Adjustment</h4>
		1)	By dragging the scroll bar for the cutoff value of distance ligation frequency at the top of the page and click &quot;Filter Run&quot; button, you can change the value of cutoff to be listed in the result table. In this tutorial, we used the criteria 2.0, in other word only interactions whose bias-remove interaction frequency is greater than two-fold of background will be listed.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_7.png" style='width:600px; '/></div><br/>
		2)	The number of entries per page can be adjusted by clicking on the drop-down button located above the interaction table. In this tutorial, we applied 25 entries per page.<br/>
		<div style="text-align: center;"><img src="images/tutorial/1_8.png" style='width:600px; '/></div><br/>
		3)	If you want to order interaction partners by a certain feature, click the header of the column. In this tutorial, interaction partners will be ordered by their bias-removed interaction frequency.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_9.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial1-4'> - Result Interpretation</h4>
		These adjustments identified interactions between rs1421085 and promoters of IRX3 and IRX5.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/1_10.png" style='width:600px; '/></div><br/>
		
	<h3 id='tutorial2'>? Session 2 : Visualization of chromatin interaction</h3>
		&nbsp;3DIV visualize the identified interactions around queried loci, which helps users to intuitively understand the interaction patterns. In this tutorial we provide an example of chr1:60,000,000 at LNCaP prostate cancer cell line and PrEC normal prostate epithelial cells. This region is well-known example of cancer-specific genome disorganization.
	
	<h4 id='tutorial2-1'> - Input parameters</h4>
		1)	Click &quot;more experiment&quot; button to load the full list of samples.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_1.png" style='width:600px; '/></div><br/>
		2)	Select LNCaP prostate cancer cell line and PrEC normal prostate epithelial cell by clicking the checkboxes.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_2.png" style='width:600px; '/></div><br/>
		3)	3DIV allows users to query gene names and SNP ID, as well as genomic coordinates. Type chr1:60000000 and click &quot;Add Sample(s)&quot; Button.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_3.png" style='width:600px; '/></div><br/>
		4)	Then, click &quot;Run&quot; button to visualize chromatin interactions around queried loci.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_4.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial2-2'> - Output</h4>
		&nbsp;Interaction visualization result have two panels which visualize chromatin interaction of LNCaP and PrEC respectively (?, ?). Each panel consists of five figures. First figure (a) is the interaction heat map, in which topologically associating domains (TAD), basic unit of genome 3D structure are indicated as the blue triangle. Second figure (b) is the one-to-all interaction frequency graph. Blue bar graph represents bias-removed chromatin interaction frequency, and magenta dots represent distance-normalized interaction frequency. Third figure (c) is the arc-diagram of the identified interactions, which are defined by the threshold displayed as the green line in figure (b). Fourth figure (d) annotates super-enhancers and RefSeq Genes. Last figure (e) describes an interaction partner of the queried loci upon clicking an interaction of interested shown in figure (c).<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_5.png" style='width:600px; '/></div><br/> 
	
	<h4 id='tutorial2-3'> - Result Adjustment</h4>
		1)	To adjust the color range of heat map, drag the scroll bar for color range above the heatmap.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_6.png" style='width:600px; '/></div><br/>
		2)	To shift visualization range without changing the queried loci, click on white space of the graph and drag horizontally towards the desired direction.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_7.png" style='width:600px; '/></div><br/>
		3)	To select other loci as the query, click the red bait indicator and drag to the loci of interest.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_8.png" style='width:600px; '/></div><br/>
		4)	To zoom-in or zoom-out visualization range, either click zoom-in/zoom-out button above the heatmap, or select the visualization range by dragging the yellow area at the top of the graph.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_9.png" style='width:600px; '/></div><br/>
		5)	By Dragging the scroll bar at the right site of interaction frequency graph, you can change significance criteria for the interactions.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_10.png" style='width:600px; '/></div><br/>
		6)	User can hide and show any subfigures by clicking &quot;heatmap open/open&quot;, &quot;Graph open/close&quot;, &quot;arc open/close&quot;, or &quot;Gene open/close&quot; button. Each button show and hide interaction frequency heat map, one-to-all interaction frequency graph, arc-representation of identified interactions, and annotations of gene body and super-enhancer respectively.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/2_11.png" style='width:600px; '/></div><br/>
		
	<h3 id='tutorial3'>? Session 3 : Comparative visualization of chromatin interaction</h3>
		&nbsp;3DIV also enables us to perform comparative analysis of gain or loss of interactions between too samples by generating a comparative heatmap with automatic scale synchronization. In this tutorial, we provide an example of chr1:60,000,000 at H1 embryonic stemcell and H1-derived mesenchymal stem cell (MSC). This region is well-known example of TAD-wise loss or gain of interaction during differentiation.
	
	<h4 id='tutorial3-1'> - Input parameters</h4>
		1)	First, choose a pair of samples by clicking the checkboxes. Click &quot;more experiment&quot; button to load the full list of samples.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_1.png" style='width:600px; '/></div><br/>
		2)	Select H1 embryonic stem cell and H1-derived Mesenchymal Stem Cell by clicking checkboxes.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_2.png" style='width:600px; '/></div><br/>
		3)	3DIV allows users to query gene names and SNP ID, as well as genomic coordinates. Type chr1:60000000 and click &quot;Add Sample(s)&quot; Button.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_3.png" style='width:600px; '/></div><br/>
		4)	Then, click &quot;run&quot; button to compare the chromatin interactions near the queried loci.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_4.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial3-2'> - Output</h4>
		The comparative interaction visualization result consists of five figures. First figure (a) is the comparative interaction heat map. Second figure (b) is the one-to-all chromatin interactions for a given query with a synchronized scale. Blue bar graph represents bias-removed chromatin interaction frequency, and magenta dots represent distance-normalized interaction frequency. Third figure (c) is the arc-diagram of identified interactions, which are defined by the threshold displayed as the green line in figure (b). Fourth figure (d) annotates super-enhancers and RefSeq Genes. Last figure (e) explains an interaction partner of the queried loci upon clicking an interaction arc of interest in figure (c).<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_5.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial3-3'> - Result Adjustment</h4>	
		1)	To adjust the color range of heat map, drag the scroll bar for color range above the heatmap.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_6.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial3-4'> - Result Interpretation</h4>	
		The comparative interaction heatmap reflects loss or gain of interactions between samples. The blue color indicates higher interaction frequencies in mesenchymal stem cell specific. In line with the heatmap, the arc-diagram of identified interactions also presents additional number of long-range chromatin interactions in MSC. <br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/3_7.png" style='width:600px; '/></div><br/>
		
	<h3 id='tutorial4'>? Session Save/Load and Visualization Export</h3>
		3DIV allows users to save and load the result, which do not require any login or registration. 3DIV makes session file which describe samples and queried locus in JSON format. User can reproduce analysis result from other computer, or share the result with collaborator by transferring the session file. Moreover, 3DIV exports figures in SVG format, which can be imported by other drawing software for building publish-ready scientific figures.<br/>
		
	<h4 id='tutorial4-1'> - Save session as a file </h4>
		1)	When user performed the analysis, click menu button, which is located at upper-right side of the panel.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_1.png" style='width:600px; '/></div><br/>
		2)	Then, click save setting button. 3DIV generate the session file, and web browser will download the session file.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_2.png" style='width:600px; '/></div><br/>
	
	<h4 id='tutorial4-2'> - Load session from a file </h4>
		1)	Click menu button, which is located at upper-right side of the panel.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_3.png" style='width:600px; '/></div><br/>
		2)	Then, click &quot;upload setting&quot; button<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_4.png" style='width:600px; '/></div><br/>
		3)	The dialogue box askes the location of file. Click &quot;choose file&quot; button and assign the location of the session file.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_5.png" style='width:600px; '/></div><br/>
		
	<h4 id='tutorial4-3'> - Export Figures</h4>	
		1)	After visualization of chromatin interactions, click &quot;save as images&quot; button. Then 3DIV generate figures in SVG format, and web browser will download the figure.<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/4_6.png" style='width:600px; '/></div><br/>
	
	
	<a id="MOVE_TOP_BTN" href="#">
 		<img src="images/top_1.png" style="width:40px;" class="top_img">
 	</a>
	
</div>


		<div id='footerTag'>
			<div class='footer-line' style="white-space: nowrap;">We highly recommend that you use the current version of Chrome and Safari browser for <b>3D-Genome Interaction Viewer & database</b></div>
			<div class='footer-line' style="white-space: nowrap;">This system has been developed by <b>KAIST</b> and <a href='http://www.kobic.re.kr' target="_blank"><b>KOBIC</b></a></div>
		</div>
		</div>
	</body>
</html>