<!DOCTYPE html>
<!-- Template by Quackit.com -->
<!-- Modified by oluwatosin oluwadare for Genome Structure Database -->
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->

    <title>GSDB</title>

    <!-- Bootstrap 4 CSS. This is for the alpha 3 release of Bootstrap 4. This should be updated when Bootstrap 4 is officially released. -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.3/css/bootstrap.min.css" integrity="sha384-MIwDKRSSImVFAZCVLtU0LMDdON6KVCrZHyVQQj6e8wIEJkW4tvwqXrbMIya1vriY" crossorigin="anonymous">
      
    <!-- Custom CSS: You can use this stylesheet to override any Bootstrap styles and/or apply your own styles -->
    <link href="css/custom.css" rel="stylesheet">
    <link rel="icon" href="images/icon.PNG">
    <!-- For icons -->
    <link href="css/font-awesome-4.6.3/css/font-awesome.min.css" rel="stylesheet">
	
	<!--- CDN DataTable-->
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/plug-ins/preview/searchPane/dataTables.searchPane.css">
	
	<!--- 3Dmol-->
	<script type="" src="https://3Dmol.csb.pitt.edu/build/3Dmol-min.js"></script> 
	<script src="https://cdn.jsdelivr.net/3dmol.js/latest/3Dmol-min.js"></script>
    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

     <!-- Navigation -->
    <nav id="topNav" class="navbar navbar-full navbar-static-top navbar-dark bg-inverse m-b-1">
        <button class="navbar-toggler hidden-md-up pull-center" type="button" data-toggle="collapse" data-target="#navbar">
            &#9776;
        </button>
        <a class="navbar-brand" href="index.php" >GSDB</a>
		<br>
        <div class="collapse navbar-toggleable-sm" id="navbar">
            <ul class="nav navbar-nav">
                <li class="nav-item">
                    <a class="nav-link" href="index.php">Home</a>
                </li>
               
			   <li class="nav-item">
                    <a class="nav-link" href="browse.php">Browse</a>
                </li>
				
				<li class="nav-item">
                    <a class="nav-link" href="softwaretools.html">Software Tools</a>
                </li>				
               
				<li class="nav-item">
                    <a class="nav-link" href="evaluate.php">Structure Evaluation</a>
                </li>
				<li class="nav-item">
                    <a class="nav-link" href="Tutorial.html">Tutorial</a>
                </li>
				 <li class="nav-item">
                    <a class="nav-link" href="contact.html">Contact Us</a>
                </li>
            </ul>
 
            </ul>
        </div>
    </nav>
    

</head>

<body>
 
    
<style>
a#MOVE_TOP_BTN {
    position: fixed;
    right: 15%;
    bottom: 200px;
    display: none;
    z-index: 999;
}
</style>

<script>
    $(function() {
    	
        $(".tutorial_btn").click(function() {
        	var id = $(this).attr('id')
        	var split = id.split('tutorial_btn');
            $('html, body').animate({
                scrollTop: $("#tutorial"+split[1]).offset().top
            }, 400);        
        });
    	
        $(window).scroll(function() {
            if ($(this).scrollTop() > 500) {
                $('#MOVE_TOP_BTN').fadeIn();
            } else {
                $('#MOVE_TOP_BTN').fadeOut();
            }
        });
        
        $("#MOVE_TOP_BTN").click(function() {
            $('html, body').animate({
                scrollTop : 0
            }, 400);
            return false;
        });
    });
</script>
 

<div class="container-fluid">
	
	<!-- Center Column -->
	<div class="col-sm-12">	
		<h3 class="card-title">	<i class="fa fa-pencil" aria-hidden="true"></i> 
				Tutorial	</h3>
		<hr>
			<div class="alert alert-success alert-dismissible" role="alert">
				<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
				 Click on the section <strong>title</strong> to view the section instructions
			</div>
		
		
<!--<div id="contents" class="subcon conwrap" style='margin-top:50px; line-height: 200%; padding: 0px 20px 20px 20px;'>-->
<div id="contents" class="subcon conwrap" >
	<div class="tutorial">
		<ol>
			<li class='tutorial_btn' id='tutorial_btn1'><b>View the Database</b>  </li>
		
			<li class='tutorial_btn' id='tutorial_btn2' style='margin-top:10px;'><b>Search the Database </b></li>
				
			<li class='tutorial_btn' id='tutorial_btn3' style='margin-top:10px;'><b>Visualization of Chromosome 3D Structure</b>  </li>
			
			<li class='tutorial_btn' id='tutorial_btn4' style='margin-top:10px;'><b>Download the Chromosome 3D Structure</b></li>
			
			<li class='tutorial_btn' id='tutorial_btn5' style='margin-top:10px;'><b>Download the Normalized Hi-C Data</b></li>
			
			<li class='tutorial_btn' id='tutorial_btn6' style='margin-top:10px;'><b>Structure Evaluation</b></li>
			
			<li class='tutorial_btn' id='tutorial_btn7' style='margin-top:10px;'><b>GSDB Computational requirement and performance</b></li>
			
			<li class='tutorial_btn' id='tutorial_btn8' style='margin-top:10px;'><b>References</b></li>

		</ol>
	</div>

		</br>	
		
		
		<h3 id='tutorial1'> Section 1 : View the Database</h3>

	 	&nbsp;Hi-C, high-throughput chromatin conformation capture (3C), is one of the most-widely used genome-wide molecular assay that can identify long-range chromatin interactions. To visualize the structure for a Hi-C data, follow the steps below<br/><br/>

		Click on <b>Browse</b> Menu in the navigation bar to load the full list of the Hi-C datasets. <b>OR</b> Click on the <b>Get Started</b> button on the homepage. Both are highlighted below
		<div style="text-align: center;"><img src="images/tutorial/search.PNG" style='width:1000px; '/></div><br/>
	
		
		<h3 id='tutorial2'> Section 2 :  Search the Database</h3>		
		<font size="5"><b> GSDB provides two ways to search the database </b></font>  <br/>
		
		<font size="5"><b>1)</b></font> GSDB provides a summary of the information provided in the database through  a <b> Summary Pane</b>. By clicking on a property/item in the Summary, the user can search the database for all the Hi-C data containing this property. <br/> <br/>
		<h4>  -  Example</h4> 
		The image below shows the Result when the user clicks on <b>100KB</b> in the <b>Resolution</b> Summary Pane<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/search_2.PNG" style='width:1000px; '/></div><br/>
		
		<font size="5"><b>2)</b></font> Users can search the database by <b>Typing</b> the keywords about the Filename,Title of Hi-C data, Hi-C data Resolution, Project Hi-C data was generated from,Project ID, and the GEO_Accesion_No  in the <b>Search Pane</b> highlighted in <font color="red"> red below </font><br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/view_db.PNG" style='width:900px; '/></div><br/>
		<h4>  - Example</h4> 
		The image below shows the Result of search for <b>1MB</b> Resolution Hi-C data<br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/search_1.PNG" style='width:1000px; '/></div><br/>
		

		
		<h3 id='tutorial3'> Section 3 :  Visualization of Chromosome 3D Structure</h3>	
		<font size="5"><b>STEP 1 :</b></font> Click on the <font size="5"><b>View</b></font> link in the 3D Structure Column to view the details and structures for a Hi-C data. <br/><br/>
	   	<div style="text-align: center;"><img src="images/tutorial/view_str.PNG" style='width:1000px;'/></div> <br/>
		
		<h4>  - Output</h4> 
		The image below shows the <b>Result</b> when a user clicks on the <b>View</b> link for the GM12878 dataset in Row 1 above</b>. 
		  <li>The <font color="red"> <b>Red Highlight </b></font> section shows the  <b>infromation about the  Resolution(s) </b> available for the Hi-C data.</li>
		  <li>The <font color="blue"> <b>Blue Highlight </b> </font> section displays the <b> structure</b> available for the Hi-C data.</li>
		  <li>The <font color="green"> <b>Green Highlight </b> </font> section shows the <b> Evaluation result</b> available for the Hi-C data. It displays the <b>Spearman Correlation</b> between the output structure and the input Hi-C data, and other evaluation result obtained.</li> 
		<div style="text-align: center;"><img src="images/tutorial/details.PNG" style='width:1000px; '/></div><br/>
		
		<font size="5"><b>STEP 2 :</b></font> Select the <b>Algorithm</b>,<b> Dataset </b>,  <b> Chromosome</b>, and Click on the <b>Display</b> button to view the structure on the viewer.<br/><br/>
		<h4>  - Example</h4> 
		The image below shows the result when a user selects  Algorithm = LorDG, Dataset = GM12878, Resolution = 1MB, Chromosome = 1. <br/><br/>
		<div style="text-align: center;"><img src="images/tutorial/gm12878_1mb.PNG" style='width:800px; '/></div><br/>
		
		<h3 id='tutorial4'> Section 4 :  Download the Chromosome 3D Structure</h3>			
		Click on the <font size="5"><b>Download</b></font> link in the <font color="green"><b>"3D Structure Column"</b> </font> to download the <b>3D structures</b> for the different algorithms available for a Hi-C data. <br/><br/>
			<div style="text-align: center;"><img src="images/tutorial/download.PNG" style='width:1000px; '/></div><br/>
			
		<h4>  - Output</h4> 
		When the user clicks on the <b>Download</b> link a compressed <b>.tar.gz</b> file with the <b>Data file Name</b>  will be downloaded.   <br/><br/>
		
		<h3 id='tutorial5'> Section 5 :  Download the Normalized Hi-C Data</h3>	
		Click on the <font size="5"><b>Download</b></font> link in the  <font color="green"><b>"Normalized Hi-C Data Column"</b> </font>  to download the <b>Normalized Hi-C Data</b> used to generate strcutures for the different algorithms. <br/><br/>
			<div style="text-align: center;"><img src="images/tutorial/download_1.PNG" style='width:1000px; '/></div><br/>
			
		<h4>  - Output</h4> 
		When the user clicks on the <b>Download</b> link a compressed <b>.tar.gz</b> file with the <b>Data file Name</b>  will be downloaded.  <br/><br/>
		
		<h3 id='tutorial6'> Section 6 : Structure Evaluation</h3>	
	   <font size="4"><b> GSDB provides two ways to evaluate a structure</b></font>  <br/>
		
		<font size="5"><b>1) Compare chromsosome structure in pdb format with input Interaction Frequency(IF) matrix </b></font> <br/>
		
		<p> To compare with IF matrix, user needs to provide the following inputs: 

		 <li>Select <b> Interaction Frequency(IF) Matrix </b>option as <b>Type of Input file 1</b></li>
		 <li><b>Provide</b> a link to a IF matrix or upload the IF matrix file from computer</li>
		 <li><b>Provide</b> a link to a chromosome structure in pdb format or upload the structure file from computer</li>
		 <li><b>Click on the Compare button </b> to compute the genome or chromsome  evaluation</li>
		<br/>
		<p><b>IF Matrix Format:</b> The data format allowed for the IF matrix is <b>a comma-seperated</b> or <b>a tab-seperated </b> n by n symmetrix matrix. <i>Where n is the number of regions/bins in the IF Matrix</i></p>
		
		<h4>  -  Example</h4> 
		Click on the Sample Inputs, <b>IF Matrix Sample input</b> for Input file 1 and <b>Structure Sample  input</b> to use them for evaluation.
		<div style="text-align: center;"><img src="images/tutorial/struct_eval.PNG" style='width:800px; '/></div><br/>
		
		
	     <h4>  - Result</h4> 
		 <p><b>On clicking the Compare button </b>, the user gets the prompt below about the evaluation result</p>
		 <div style="text-align: center;"><img src="images/tutorial/eval_result.PNG" style='width:700px; '/></div>
		 
		</br> 		
		<font size="5"><b>2) Compare a chromsome structure with another chromsome structure in pdb format</b></font> <br/>
		
		<p> To compare two structures, user needs to provide the following inputs: 

		 <li>Select<b> Chromosome structure in PDB format </b>option as <b>Type of Input file 1</b> </li>
		 <li><b>Provide</b> a link to a Chromosome structure  or upload the Structure file from computer</li>
		 <li><b>Provide</b> a link to a another  Chromosome structure or upload the Structure file from computer</li>
		 <li><b>Click on the Compare </b> Button to compute the genome or chromsome evaluation</li>
			
		<h4>  -  Example</h4> 
		Click on the Sample Inputs, <b>Structure Sample input</b> for Input file 1 and <b>Structure Sample input</b> to use them for evaluation.
		<div style="text-align: center;"><img src="images/tutorial/struc_struc.PNG" style='width:800px; '/></div><br/>
		
		
	     <h4>  - Result</h4> 
		 <p><b>On clicking the Compare button </b>, the user gets the prompt below about the evaluation result</p>
		 <div style="text-align: center;"><img src="images/tutorial/eval_result_1.PNG" style='width:700px; '/></div>
		 
		</br> </br> 
		
		
		<h3 id='tutorial7'> Section 7 :  Computational requirement and performance</h3>	
		
		The GSDB chromosome structure generation was done on two server machines: a x86_64 bit Redhat-Linux server con-sisting of multi-core Intel(R) Xeon(R) CPU E7-L8867 @ 2.13GHz with 120 GB RAM and a high-performance computing cluster (Lewis) with Linux. 
		Using a high-performance computing (HPC) cluster machine, we allocated 10 cores, 80G of memory, with a time limit of 2 days for each chromosome structure reconstruction task per algorithm. No structures are avilable for chromosome Hi-C data that took more than two days to be constructed by an algorithm. 
		Find more details about Lewis Cluster <a href="http://docs.rnet.missouri.edu/Policy/partition-policy"style="color: rgb(0,0,255)"> here</a> 		 
		
	     

		</br> </br> </br>
		
	
		<h3 id='tutorial8'> Section 8 :  References</h3>	
		
	  <li> Wang, S., Xu, J. and Zeng, J., 2015. Inferential modeling of 3D chromatin structure. Nucleic acids research, 43(8), pp.e54-e54.</li>
	  <li> Varoquaux, N., Ay, F., Noble, W.S. and Vert, J.P., 2014. A statistical approach for inferring the 3D structure of the genome. Bioinformatics, 30(12), pp.i26-i33.</li>	 
	  <li> Zhu, G., Deng, W., Hu, H., Ma, R., Zhang, S., Yang, J., Peng, J., Kaplan, T. and Zeng, J., 2018. Reconstructing spatial organizations of chromosomes through manifold learning. Nucleic acids research, 46(8), pp.e50-e50.</li>
	  <li> Rieber, L. and Mahony, S., 2017. miniMDS: 3D structural inference from high-resolution Hi-C data. Bioinformatics, 33(14), pp.i261-i266.</li>
	  <li> Zhang, Z., Li, G., Toh, K.C. and Sung, W.K., 2013. 3D chromosome modeling with semi-definite programming and Hi-C data. Journal of computational biology, 20(11), pp.831-846.</li>
	  <li> Zou, C., Zhang, Y. and Ouyang, Z., 2016. HSA: integrating multi-track Hi-C data for genome-scale reconstruction of 3D chromatin structure. Genome biology, 17(1), p.40.</li>
	  <li> Lesne, A., Riposo, J., Roger, P., Cournac, A. and Mozziconacci, J., 2014. 3D genome reconstruction from chromosomal contacts. Nature methods, 11(11), p.1141.</li>
	  <li> O. Oluwadare; Y. Zhang; J. Cheng. A maximum likelihood algorithm for reconstructing 3D structures of human chromosomes from chromosomal contact data, BMC Genomics, 19:161, 2018 <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4546-8">[at BMC Genomics]</a></li>
	  <li> T. Trieu, J. Cheng. 3D Genome Structure Modeling by Lorentzian Objective Function. Nucleic Acids Research, accepted, 2016.  <a href="https://academic.oup.com/nar/article/45/3/1049/2605802">[at NAR]</a></li>
	  <li> B. Adhikari, T. Tuan, J. Cheng. Chromosome3D: Reconstructing Three-Dimensional Chromosomal Structures from Hi-C Interaction Frequency Data using Distance Geometry Simulated Annealing. BMC Genomics, 17:886, 2016.  <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3210-4">[at BMC Genomics]</a></li>
	  <li> T. Tuan, J. Cheng. MOGEN: a tool for reconstructing 3D models of genomes from chromosomal conformation capturing data. Bioinformatics, accepted, 2015. doi: 10.1093/bioinformatics/btv754. <a href="https://academic.oup.com/bioinformatics/article/32/9/1286/1743926">[at BMC Bioinformatics]</a></li>
	  <li> Rosenthal, M., Bryner, D., Huffer, F., Evans, S., Srivastava, A. and Neretti, N., 2018. Bayesian Estimation of 3D Chromosomal Structure from Single Cell Hi-C Data. bioRxiv, p.316265.</li>
		 
		
	</div><!--/Center Column-->

	<p style="text-align:center">
		<img src="https://relay.hekademeia.org/track/image?src=hek-host-wc" alt="Hosting by Hekademeia" style="width: 160px;">
	</p>
		
	
</div><!--/container-fluid-->

	
    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!-- jQuery library -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.0.0/jquery.min.js" integrity="sha384-THPy051/pYDQGanwU6poAc/hOdQxjnOEXzbT+OuUAFqNqFjL+4IGLBgCJC3ZOShY" crossorigin="anonymous"></script>

    <!-- Tether -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.2.0/js/tether.min.js" integrity="sha384-Plbmg8JY28KFelvJVai01l8WyZzrYWG825m+cZ0eDDS1f7d/js6ikvy1+X+guPIB" crossorigin="anonymous"></script>

    <!-- Bootstrap 4 JavaScript. This is for the alpha 3 release of Bootstrap 4. This should be updated when Bootstrap 4 is officially released. -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.3/js/bootstrap.min.js" integrity="sha384-ux8v3A6CPtOTqOzMKiuo3d/DomGaaClxFYdCu2HPMBEkf6x2xiDyJ7gkXU0MWwaD" crossorigin="anonymous"></script>

	<!----CDN DataTable -->
	<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.16/js/jquery.dataTables.js"></script>
	<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/plug-ins/preview/searchPane/dataTables.searchPane.js"></script>
	
	
		
	
	
</body>

</html>
